src/main/resources/edu/cornell/kfs/module/purap/document/workflow/RequisitionDocument.xml

```diff
@@ -45,6 +45,7 @@
 				<status>In Process</status>
 				<status>Awaiting C and G Approval</status>
 				<status>Disapproved Commodity Review</status>
+				<status>Disapproved Ad Hoc</status>
         <status>Disapproved AdHoc</status>
```

Comment:
It looks like we had already added an ad hoc disapproval status similar to the KualiCo one, although ours has "AdHoc" instead of "Ad Hoc" in the name. Do we need to remove our CU-specific status for ad hoc disapproval, and do we need more code and SQL changes if so?